sources/packages/web/src/components/aest/institution/modals/ApproveProgramModal.vue

```diff
@@ -12,12 +12,14 @@
       ></formio>
     </template>
     <template v-slot:footer>
-      <v-btn variant="outlined" :color="COLOR_BLUE" @click="dialogClosed">
-        Cancel
-      </v-btn>
-      <v-btn class="primary-btn-background" @click="approveProgram">
-        Approve now
-      </v-btn>
+      <v-row class="m-0 p-0">
```

Comment:
This can be replaced by the `footer-buttons` and we can avoid codes like `color="COLOR_BLUE"`.

Reply:
ya... I was planning to do these changes in my next PR.